client/my-sites/people/invite-people/index.jsx

```diff
@@ -89,8 +93,18 @@ const InvitePeople = React.createClass( {
 			return value;
 		} );
 
+		const filteredErrors = pickBy( errors, ( error, key ) => {
```

Comment:
We could. Which would make it a bit easier to use `filter` for both errors and successes in the case above.

But, it would also slightly complicate the error logic in `getTokensWithStatus()`, which is currently: `if ( errors && errors[ value ] )`.

That would probably become something like `if ( some( errors, ( error ) => { error.value === value } ) )`.

Which is still readable, but I'm not sure that it's worth the effort to change.